src/map/status.cpp

```diff
@@ -10171,6 +10181,11 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 	if( !sc )
 		return 0; // Unable to receive status changes
 
+	// Scripted status changes only work for players for the time being
+	if( scdb->script != nullptr && bl->type != BL_PC ){
```

Comment:
```suggestion
	if( scdb->script != nullptr && bl->type != BL_PC ){
		ShowError( "status_change_start: Trying to start the scripted status change %d on a non player.\n", type );
```
Like this?

Reply:
Yes, maybe "failed" would be more explicit than "trying" tho